tika-bundle/pom.xml

```diff
@@ -164,6 +164,7 @@
                             tika-parsers;inline=true,
                             commons-compress,
                             xz,
+                            asm,
```

Comment:
Please include the specific asm dependency in the pom file so a different version of it doesn't get pulled in inadvertently in the future if transitive dependencies change.